### ISSUE TITLE
[BUG] GUI redirects me to invalid url

### ISSUE THREAD

**Issue by mtzirkel** [andrew-hossack/dash-tools#65]
**Describe your context**
I am trying to host my dash app on render. I can go through the gui checklist but when I hip deploy to render it redirects me to render and I see the following message 
![image](https://user-images.githubusercontent.com/16643754/197814372-3cce5d2b-9690-492f-a48c-af48b00fd79d.png)

- replace the result of `dash-tools --version` below

```
dashtools 1.10.1

```

- please tell us what operating system you are using

```
linux POPos
```

**Describe the bug**

It seems to me that there is some path concatenation that is not working correctly and in quickly scanning your repository I believe that on line 27 of [deployPage.py ](https://github.com/andrew-hossack/dash-tools/blob/main/src/dashtools/dashboard/pages/deployPage.py) is where this is occurring. 
When looking at my github i see that dashtools is successfully pushing to my github repo(DashTool Automatic Push - 10/25/2022 @ 9:01:44)

I created a pull request that might work. Not sure what else uses that function though

**Expected behavior**

I expect it to push to render

Thanks for your work this is a great repo

**Comment by andrew-hossack** [comment]
Hey @mtzirkel, for some reason I'm not seeing that issue on my laptop. What steps did you take to init and add your Git repo?

**Comment by mtzirkel** [comment]
Initially i was using Pycharm's version control tool but I was able to recreate it using the gihub CLI in another repository. Both have the same issue of redirecting me to https://github.com///github.com/mtzirkel/dashtemp. I was able to use the yaml file and requirements.txt and get it on render earlier following their setup instructions. For what it is worth.

**Comment by andrew-hossack** [comment]
Strange, I will test this more and release a patch soon. Would you mind running the following command in the application you have pushed to github and sharing the output?

```
git config --get remote.origin.url
```

It should read something like **git@github.com:andrew-hossack/testrepo2.git**

**Comment by andrew-hossack** [comment]
I'm pretty sure I know what the bug is. Only SSH keys for remote origins are supported right now, eg. if you added your remote origin via HTTP it won't work. Let me push out a new update to address this.

**Comment by andrew-hossack** [comment]
I opened https://github.com/andrew-hossack/dash-tools/pull/67, feel free to review!

**Comment by andrew-hossack** [comment]
I pushed out the fix to Version 1.10.2, so go ahead and run 

```
python -m pip install --upgrade dash-tools
```

and try again. Let me know if you still run into issues.
